tests/common/secure_sockets/aws_test_tcp.c

```diff
@@ -1956,7 +1956,7 @@ TEST( Full_TCP, AFQP_SOCKETS_Connect_InvalidAddressLength )
     prvConnect_InvalidAddressLength( eNonsecure, 0 );
 
     /* AddressLength 1000. */
-    prvConnect_InvalidAddressLength( eNonsecure, 1000 );
+    prvConnect_InvalidAddressLength( eNonsecure, 100 );
```

Comment:
255 maybe lol

Reply:
and also:
- /* AddressLength 1000. */
+ /* AddressLength 100. */
?

Reply:
Thanks. Will update the comment too.